src/components/apiDelegation/given/OverviewPage/OverviewPage.tsx

```diff
@@ -8,13 +8,17 @@ import { OverviewPageContent } from '../../reusables/OverviewPageContent';
 import { LayoutState } from '../../reusables/LayoutState';
 import { PageContainer } from '../../../reusables/PageContainer';
 
+import classes from './OverviewPage.module.css';
+
 export const OverviewPage = () => {
   const { t } = useTranslation('common');
 
   return (
     <PageContainer>
       <Page>
-        <PageHeader icon={<ApiIcon />}>{t('api_delegation.api_delegations')}</PageHeader>
+        <div className={classes.pageHeaderWrapper}>
+          <PageHeader icon={<ApiIcon />}>{t('api_delegation.api_delegations')}</PageHeader>
+        </div>
         <PageContent>
```

Comment:
Have you pulled the most recent main?
If so then you can use the .pageContent class from '@/main.module.css' to get more responsive margins for the page.
We need all the space we can get on mobile screens and the standard margin of 3rem wastes a lot of that space. Smaller margins give us more to go on and is more in line with the designs for the issue.

Reply:
"Have you pulled the most recent main?... " etc.

Helt enig i alt dette. Men det redder ikke Titanic her. Tror jeg da. Men, som sagt, skal prate med Albert om det.

Reply:
Her er forresten log fra git, siste pull var ca 5 timer før PR ble sendt inn:

"commit 0b34211b47477d055438c058c8cdca210bae02ad
Author: Torgeir Holen <torgeir333@gmail.com>
Date:   Thu Feb 2 13:18:26 2023 +0100

    wrap PageHeader for css

commit 655eedbb0d5567d8198ac4e9032dc9f1ec31169f (origin/main, origin/HEAD, main_error_020223, main)
Author: Albertlarsen <31471142+Albertlarsen@users.noreply.github.com>
Date:   Thu Feb 2 08:56:42 2023 +0100
"

Reply:
Nei, det er ikke sånn at denne ene tingen er nok til å redde alt, men alle monner drar.
Så jeg synes det fremdeles er viktig å gjøre de takene man kan.
Dette er basert på en kommentar jeg hadde på Albert sin tilsvarende issue tidligere og da ble vi enige om at dette var en lur ting å gjøre, så jeg vil vel tro at han fremdeles er enig i det 🙂